api/types/client.go

```diff
@@ -114,8 +114,9 @@ type NetworkListOptions struct {
 
 // HijackedResponse holds connection information for a hijacked request.
 type HijackedResponse struct {
-	Conn   net.Conn
-	Reader *bufio.Reader
+	Multiplexed *bool
```

Comment:
Maybe the content type string here?

Reply:
That's fine for me (or a function in the package that defines the content-type values)... and/or a generic `ConsumeStdStream`.

I like to avoid such bool fields because invariably more than the bool is needed, and then we are dealing with pointers everywhere.